src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import pages.LoginPage;
+import pages.WebDriverInit;
+
+import java.io.File;
+
+public class BaseTest {
+    private static final String SCREENSHOT_FOLDER_PATH = "/Users/costicadragu/Desktop/screenshots/";
```

Comment:
it is better to save it somewhere in the target folder of the project `/target/screenshots/`